relay-server/src/service.rs

```diff
@@ -194,8 +206,9 @@ impl ServiceState {
         let health_check = HealthCheckService::new(config.clone()).start();
         let relay_cache = RelayCacheService::new(config.clone()).start();
 
-        let aggregator = Aggregator::new(config.aggregator_config(), project_cache.recipient());
-        registry.set(Arbiter::start(|_| aggregator));
+        let aggregator =
+            AggregatorService::new(config.aggregator_config(), project_cache.recipient());
+        let aggregator = aggregator.start();
```

Comment:
Please start the aggregator in a dedicated runtime for the time being. It is resource-intensive, mostly through allocations. Until we update its internals to use `spawn_blocking`, it is safer to follow the current thread allocation and have it in a dedicated, single-threaded runtime. 

See `outcome_runtime` for an example.